helm/vitess/values.yaml

```diff
@@ -14,6 +14,10 @@
 # will be taken from defaults defined below.
 # topology:
 
+# Repository for vitess images. Defaults to vitess repo on dockerhub. 
+# Override by specifying the full path of alternate repository, e.g. <give an example>
```

Comment:
Can you give a real example here? I couldn't think of a valid one.

Reply:
any other changes

Reply:
I'll defer to @derekperkins for approval.